### DISCUSSION TITLE
“#FF797979”不是属性“Color”的有效值。

### DISCUSSION THREAD

**Discussion by stargazerwh** [HandyOrg/HandyControl#1581] [Q&A]
只要添加Button或者ToggleButton就会报错

调用堆栈 	
 WindowsBase.dll!System.Windows.DependencyObject.GetEffectiveValue(System.Windows.EntryIndex entryIndex, System.Windows.DependencyProperty dp, System.Windows.RequestFlags requests)	

HandyControl.dll!HandyControl.Controls.SimplePanel.ArrangeOverride(System.Windows.Size arrangeSize)

**Comment by ReverseLw** [comment]
https://www.v-club.com/qAnswers/2P4VwVfZ27

**Reply by ReverseLw** [reply]
我可能找到我自已遇到的问题了，我调用了海康相机的第三方库。可以与HandyControl起冲突了

**Reply by zzzlqqq123** [reply]
兄弟，我也遇到了这个问题，最后怎么解决的？
